lib/Models/GltfCatalogItem.ts

```diff
@@ -18,6 +17,11 @@ import Transforms from "terriajs-cesium/Source/Core/Transforms";
 import HeightReference from "terriajs-cesium/Source/Scene/HeightReference";
 import CommonStrata from "./CommonStrata";
 
+// We want TS to look at the type declared in lib/ThirdParty/terriajs-cesium-extra/index.d.ts
+// and import doesn't allows us to do that, so instead we use require + type casting to ensure
+// we still maintain the type checking, without TS screaming with errors
+const Axis: Axis = require("terriajs-cesium/Source/Scene/Axis").default;
```

Comment:
I think a decision to import things like this might warrant an ADR. In future we might restructure how terriajs & terriajs-cesium work together and it'd be good to have the work you've done and solutions you've tried documented.

Reply:
Yup that's fair, I created a draft ADR